components/header.tsx

```diff
@@ -19,14 +22,16 @@ export default function Header({ userName }: HeaderProps): JSX.Element {
               </a>
             </Link>
           </h1>
-          {userName && (
+          {user.loggedIn ? (
```

Comment:
We'll probably need to move the header in to each respective layout (resident/staff) since we have conflicting use cases here for what a user is.